core/src/main/java/org/mskcc/cbio/oncokb/model/Query.java

```diff
@@ -154,7 +155,7 @@ public void setAlteration(String alteration) {
         if (alteration != null) {
             alteration = alteration.replace("p.", "");
         }
-        this.alteration = alteration;
+        this.alteration = AminoAcidConverterUtils.resolveHgvspShortFromHgvsp(alteration);;
```

Comment:
Add some tests in IndicatorUtilsTest to verify three letters variants are supported.